src/vmm/src/memory_snapshot.rs

```diff
@@ -117,8 +121,11 @@ impl SnapshotMemory for GuestMemoryMmap {
         writer: &mut T,
         dirty_bitmap: &DirtyBitmap,
     ) -> std::result::Result<(), Error> {
-        let page_size = sysconf::page::pagesize();
         let mut writer_offset = 0;
+        let page_size = match unsafe { libc::sysconf(libc::_SC_PAGESIZE) } {
```

Comment:
I suggest to write a function for this and use that in the unit tests, WDYT?

Reply:
Good idea!